lib/ufe/CMakeLists.txt

```diff
@@ -0,0 +1,207 @@
+# -----------------------------------------------------------------------------
```

Comment:
I'm assuming that this block was just copied from the other cmake file and nothing has really changed in it. If something did change can you please point it out?

Reply:
Correct. Nothing really changed here, expect the sub-directory prefix that is gone now:

https://github.com/Autodesk/maya-usd/blob/334926c3a6e965fcea048fb76f3980cf50e3f002/lib/CMakeLists.txt#L80